packages/preview2-shim/lib/http/wasi-http.js

```diff
@@ -2,18 +2,18 @@
 // https://github.com/bytecodealliance/wasmtime/blob/8efcb9851602287fd07a1a1e91501f51f2653d7e/crates/wasi-http/
 
 /**
- * @typedef {import("../../types/imports/wasi-http-types").Fields} Fields
```

Comment:
Curious why this didn't regenerate with the script previously?

Reply:
i don't think this file is update by the scripts at all. there is probably some way to get the types from the entrypoint instead, but i could not get it to work a while back